examples/swap_networks.py

```diff
@@ -0,0 +1,83 @@
+"""Demonstrates swap networks."""
+
+import itertools
+import random
+
+import cirq
+import cirq.contrib.acquaintance as cca
+
+
+def get_random_graph(n_vertices, edge_prob=0.5):
+    return [
+        ij for ij in itertools.combinations(range(n_vertices), 2)
+        if random.random() <= edge_prob
+    ]
+
+
+def get_max_cut_qaoa_circuit(n_vertices, edges, beta, gamma):
+    """Implement a single round of QAOA for MaxCut using linearly connected
+    qubits.
+
+    Args:
+        n_vertices: The number of vertices.
+        edges: The list of edges.
+        beta: The mixing angle.
+        gamma: The phase separation angle.
+
+    Returns:
+        A circuit implementing a single round of QAOA with the specified
+        angles.
+    """
+
+    ## G_{i,j} ∝ exp(i gamma (|01><01| + |10><10|))
```

Comment:
I'm not familiar with this double comment `##` convention.

Reply:
That's a very diplomatic way of pointing out a typo :)